chainer/variable.py

```diff
@@ -313,6 +313,16 @@ def _set_grad_var_if_available(self, g):
         if var is not None:
             var._set_grad_var_without_check(g)
 
+    def _pop_grad_var_if_available(self):
```

Comment:
The meaning of `_if_available` is not clear for me.
Perhaps `_if_alive` is better, or perhaps it can simply be removed.